modules/SofaGraphComponent/SceneCheckerVisitor.cpp

```diff
@@ -62,7 +62,7 @@ void SceneCheckerVisitor::addHookInChangeSet(const std::string& version, ChangeS
 void SceneCheckerVisitor::installChangeSets()
 {
     addHookInChangeSet("17.06", [](Base* o){
-        if(o->getClassName() == "RestShapeSpringsForceField" && o->findData("external_rest_shape")->isSet())
+        if(o->getClassName() == "RestShapeSpringsForceField" && o->findLink("external_rest_shape")->getSize() != 0)
             msg_warning(o) << "RestShapeSpringsForceField have changed since 17.06. The parameter 'external_rest_shape' is now a Link. To fix your scene you need to add and '@' in front of the provided path. See PR#315" ;
```

Comment:
Just reading the code, it seems to me that we really want to search for a **Data** (and raise an warning if we find one).
@damienmarchal ?